exercises/wordy/canonical-data.json

```diff
@@ -134,6 +134,38 @@
         "question": "Who is the President of the United States?"
       },
       "expected": false
```

Comment:
I think all instances of `"expected": false` should be converted to syntax errors in this PR.

Reply:
:+1: 

will happen when rebasing on top of https://github.com/exercism/problem-specifications/pull/1334